gammapy/scripts/config/docs.yaml

```diff
@@ -76,6 +75,17 @@ datasets:
               nbin: 73
               unit: TeV
               interp: log
+    geom-irf:
```

Comment:
@Bultako I think it's clearer if the section in the config file which declares the geometry, directly corresponds to `MapDataset.create()` like so:
```
   geom:
        skydir: [83.633, 22.014]
        width: [5, 5]
        binsz: 0.02
        coordsys: CEL
        proj: TAN
        axes:
          - name: energy
            hi_bnd: 10
            lo_bnd: 1
            nbin: 5
            interp: log
            node_type: edges
            unit: TeV
    energy-axis-true:
          - name: energy
            hi_bnd: 10
            lo_bnd: 1
            nbin: 5
            interp: log
            node_type: edges
            unit: TeV
    binsz-irf: 0.2 deg
    margin-irf: 0.5 deg
```
The additional `geom-irf` section could be removed.

Reply:
Thanks @adonath 

I've addressed your comments in last commit https://github.com/gammapy/gammapy/pull/2479/commits/d936eb93f4a7817b4ea1cee387a7095386d86c8c The ``hess.ipynb`` notebook still needs the values for true energy as params so to not break.